src/server/scripts/Kalimdor/zone_mulgore.cpp

```diff
@@ -77,20 +78,21 @@ class npc_agitated_earth_spirit : public CreatureScript
             {
                 switch (urand(0, 1))
                 {
-                case 0:
-                {
-                    me->setFaction(35);
-                    if (Player* player = ObjectAccessor::GetPlayer(*me, _playerGUID))
-                        player->KilledMonsterCredit(NPC_EARTH_SPIRIT_CREDIT_BUNNY);
-
-                    me->DespawnOrUnsummon(1000);
-                    break;
-                }
-                case 1:
-                    me->setFaction(14);
-                    if (Player* player = ObjectAccessor::GetPlayer(*me, _playerGUID))
-                        AttackStart(player);
-                    break;
+                    case 0:
+                    {
+                        me->setFaction(35);
+                        if (Player* player = ObjectAccessor::GetPlayer(*me, playerGUID))
+                            player->KilledMonsterCredit(NPC_EARTH_SPIRIT_CREDIT_BUNNY);
+
+                        me->DespawnOrUnsummon(1000);
+                        break;
+                    }
+                    case 1:
+                        Talk(0);
```

Comment:
this wont work, the type of the message in db is 42 currently in skyfire it is (CHAT_MSG_RAID_BOSS_WHISPER) for that message you need to set the target in this case its player.

Talk(0, player);